x-pack/plugins/task_manager/server/task_scheduling.ts

```diff
@@ -99,7 +97,6 @@ export class TaskScheduling {
   constructor(opts: TaskSchedulingOpts) {
     this.logger = opts.logger;
     this.middleware = opts.middleware;
-    this.taskPollingLifecycle = opts.taskPollingLifecycle;
```

Comment:
I think this was more like a clean up. Since we don't use taskPollingLifecycle for runSoon, listening for its events is not necessary.